### ISSUE TITLE
Example config-sorting-duplicates.yaml: descriptions of what is returned

### ISSUE THREAD

**Issue by thomas-delva** [KNowledgeOnWebScale/walder#32]
# Description
I will admit this is a nitpick, but still. In [the readme](https://github.com/KNowledgeOnWebScale/walder#options) the return types for the routes for the `config-sorting-duplicates.yaml` example do not match what is actually returned:

* For <http://localhost:3000/music/Marcella%20Detroit/no_duplicates> and <http://localhost:3000/music/Marcella%20Detroit/everything_together> it says "songs with their artists", but only song names are returned.
* For <http://localhost:3000/artist/David%20Bowie> it says "song names and movie URLs", but only movie URLs are returned.

As far as I can tell, the sorting and duplicate removal *is done correctly* though.

# How to reproduce
1. Start Walder version 2.0.2 with the command `yarn start -c example/config-sorting-duplicates.yaml -l debug --no-cache`
2. Surf to <http://localhost:3000/music/Marcella%20Detroit/no_duplicates>, <http://localhost:3000/music/Marcella%20Detroit/everything_together> or <http://localhost:3000/artist/David%20Bowie>

**Comment by pheyvaer** [comment]
Regarding http://localhost:3000/artist/David%20Bowie I get the a list of URLs first and then a list of songs. Did you scroll down on the page?

**Comment by pheyvaer** [comment]
@thomas-delva Bump 😉

**Comment by thomas-delva** [comment]
Yes, regarding http://localhost:3000/artist/David%20Bowie, the description is indeed correct! I read it as "song names *with* movie URLs" in error.

Leaving the issue open wrt the first bullet point
